### PR TITLE
LowPtElectrons: ID models for 10_6_X

### PR DESCRIPTION
**Pull request by bainbrid**
This PR contains models that are required by PR https://github.com/cms-sw/cmssw/pull/32372 (which is a back port of https://github.com/cms-sw/cmssw/pull/31220).

There are two weights files:

`RunII_Autumn18_LowPtElectrons_mva_id.xml.gz`
- this is the model used by RECO throughout the 10_6_X release cycle 

`LowPtElectrons_ID_2020Sept15.root`
- this is the model to be used for reminiAOD UL (via the run2_miniAOD_UL modifier)
- is the same model used in https://github.com/cms-sw/cmssw/pull/31220
- has depth=15, trees=1000, extra variables, no regression, base on BParking samples.

### CODE REVIEW

**Comment by slava77** [comment]
I'm confused, what is the difference between this PR and #15? 

Both are a single commit and both add a file LowPtElectrons/LowPtElectrons_ID_2020Sept15.root
... and #15 is already merged

**Comment by bainbrid** [comment]
In response to: https://github.com/cms-data/RecoEgamma-ElectronIdentification/pull/17#issuecomment-739555474:

In short, [this](https://github.com/CMSBParking/RecoEgamma-ElectronIdentification/commit/2aa9f047d476815fac40c5b6a106983bbdf46c2d#diff-72d9ce87b279f68129c5861a9448a971a70c5b38c07373005980eadf0a990d8b) commit changed the `RunII_Autumn18_LowPtElectrons_mva_id.xml.gz` file. Here's the [history](https://github.com/cms-data/RecoEgamma-ElectronIdentification/commits/master/LowPtElectrons/RunII_Autumn18_LowPtElectrons_mva_id.xml.gz) for the XML file. This PR reverts that change.  

It's difficult to know how to navigate GitHub to find the commit or tag used to build a particular release. I've tried to navigate the history as best I can, details below. I think they are correct. 

---

The two PRs are branched from a different base, and:
- they contain different versions of `RunII_Autumn18_LowPtElectrons_mva_id.xml.gz`
- while `LowPtElectrons_ID_2020Sept15.root` is identical

1) PR #15 is based on the branch `CMSBParking:from-CMSSW_10_2_15_2020Sept15`

```
> git log --oneline RunII_Autumn18_LowPtElectrons_mva_id.xml.gz
2aa9f04 Extra variables for LowPtElectrons Id added. Depth10, no regression
c79bcd5 added Autumn18 trainings
```

```
> git diff 2aa9f04^ 2aa9f04
diff --git a/LowPtElectrons/RunII_Autumn18_LowPtElectrons_mva_id.xml.gz b/LowPtElectrons/RunII_Autumn18_LowPtElectrons_mva_id.xml.gz
index 5c15cb1..29ce4ac 100644
```

Here is the diff on GitHub: https://github.com/CMSBParking/RecoEgamma-ElectronIdentification/commit/2aa9f047d476815fac40c5b6a106983bbdf46c2d#diff-72d9ce87b279f68129c5861a9448a971a70c5b38c07373005980eadf0a990d8b.

At the head of this branch, the size of `RunII_Autumn18_LowPtElectrons_mva_id.xml.gz` is 8519842 bytes.

2) PT #17 (this PR) is based on the branch `CMSBParking:LowPtElectrons_ID_2020Sept15_for106X`

```
> git log --oneline RunII_Autumn18_LowPtElectrons_mva_id.xml.gz gives:
c79bcd5 added Autumn18 trainings
```

At the head of this branch, the size of `RunII_Autumn18_LowPtElectrons_mva_id.xml.gz` is 7599273 bytes.

**Comment by slava77** [comment]
I still could not follow the description given what's showing up in the web interface: only LowPtElectrons/LowPtElectrons_ID_2020Sept15.root is changed in this PR, while the description is about RunII_Autumn18_LowPtElectrons_mva_id.xml.gz

**Comment by bainbrid** [comment]
This is the commit history for the branch for this PR (#17):
https://github.com/CMSBParking/RecoEgamma-ElectronIdentification/commits/from-CMSSW_10_2_15_2020Sept15
- The penultimate commit is a merge from PR #13 (dated Feb 2019), which contains the xml.gz file used throughout the 10_6_X cycle. Importantly, this is the file we need to maintain backward compatibility for the RECO sequence in 10_6_X. 
- The last commit is the one I've just made, which adds the 2020Sept15 model for reminiAOD.

This is the commit history for the branch for PR #15: 
https://github.com/CMSBParking/RecoEgamma-ElectronIdentification/commits/from-CMSSW_10_2_15_2020Sept15
- The penultimate commit (https://github.com/CMSBParking/RecoEgamma-ElectronIdentification/commit/4379aec9141541e8f49e841efb5f37ac267e0a2f) is a merge from PR #14 (dated Aug 2020), which contains an _update_ to the xml.gz file, which _differs_ from the model throughout the 10_6_X cycle. Importantly, this is the file we need to _revert_ to maintain backward compatibility for the RECO sequence in 10_6_X. 
- The last commit adds the 2020Sept15 model, which was recently merged for https://github.com/cms-sw/cmssw/pull/31220

This is the commit history for the master of cms-data:
https://github.com/cms-data/RecoEgamma-ElectronIdentification/commits/master
- In short, we want to revert the change in commit https://github.com/cms-data/RecoEgamma-ElectronIdentification/commit/2aa9f047d476815fac40c5b6a106983bbdf46c2d

**Comment by slava77** [comment]
> This is the commit history for the branch for this PR (#17):

all of this apparently means that we 
- need a PR to a branch corresponding to 10_6_X, which does not exist
- or we need to explicitly revert the update in the xml file in the master (remake this PR on top of the master)

**Comment by slava77** [comment]
@smuzaffar @mrodozov 
we will need a separate branch for this external for the update to 10_6_X.

I see that the current version in 10_6_X is V01-01-03,
perhaps V01-01-03_X  would work, but feel free to suggest another branch name (and go ahead to make it, please).

**Comment by smuzaffar** [comment]
@slava77 , I created a new brack V01-01-XX (based on tag V01-01-03)  and updated this PR to be based on it.
If this PR is good to go then we can merge and create a tag V01-01-04.

**Comment by slava77** [comment]
> @slava77 , I created a new branch V01-01-XX (based on tag V01-01-03) and updated this PR to be based on it.
> If this PR is good to go then we can merge and create a tag V01-01-04.

Thank you.
Let me see what comes out from tests with #32372

**Comment by slava77** [comment]
+1

> > @slava77 , I created a new branch V01-01-XX (based on tag V01-01-03) and updated this PR to be based on it.
> > If this PR is good to go then we can merge and create a tag V01-01-04.
> 
> Thank you.
> Let me see what comes out from tests with [cms-sw/cmssw#32372](https://github.com/cms-sw/cmssw/pull/32372)

I think that the last set of tests was good and we can go ahead with merging this PR.

**Comment by mrodozov** [comment]
+externals
